installer.sh

```diff
@@ -297,238 +60,6 @@ FIXER_URL="https://iobroker.net/fix.sh"
 # Remember the full path of bash
 BASH_CMDLINE=$(which bash)
 
```

Comment:
I guess this and the following lines above could also be moved to the lib, what do you think?
```
# Check if "sudo" command is available (in case we're not root)
if [ "$IS_ROOT" != true ]; then
	if [[ $(which "sudo" 2>/dev/null) != *"/sudo" ]]; then
		echo "${red}Cannot continue because the \"sudo\" command is not available!${normal}"
		echo "Please install it first using \"$INSTALL_CMD install sudo\""
		exit 1
	fi
fi
```
I mean its unlikely that sudo is missing when ioBroker is already installed, but it doesn't hurt either.

```
CONTROLLER_DIR="$IOB_DIR/node_modules/iobroker.js-controller"
INSTALLER_INFO_FILE="$IOB_DIR/INSTALLER_INFO.txt"
```

```

# Where the fixer script is located
FIXER_URL="https://iobroker.net/fix.sh"
```

Reply:
Well, there are several other parts that could be extracted to the LIB.
For now, i would like to have just `functions` in the LIB.
Lets do the rest later. We have done a lot of changes and i want to be sure that they run fine in production before we proceed...

Reply:
Ok. I moved this few lines to the lib.